@trycourier/courier-ui-inbox/src/types/courier-inbox-theme-manager.ts

```diff
@@ -105,6 +105,10 @@ export class CourierInboxThemeManager {
     this.updateTheme();
   }
 
+  public get mode(): CourierComponentThemeMode {
```

Comment:
Optional: Could also use the corresponding `set mode(mode: CourierComponentThemeMode) { ... }` signature above